cds_migrator_kit/rdm/migration/transform/models/thesis.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of CERN Document Server.
+# Copyright (C) 2024 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""CDS-RDM Summer student model."""
+
+from .base import model as base_model
+from .overdo import CdsOverdo
+
+
+class ThesisModel(CdsOverdo):
+    """Translation Index for CERN Summer Student Project Notes."""
+
+    # __query__ = "980__:INTNOTEEPPUBLL 980__:NOTE 037__:CERN-STUDENTS-Note-\"/(.*?)/\""
+    __query__ = "(980__:THESIS OR 980__:Thesis OR 980__:thesis) -980__c:HIDDEN"
```

Comment:
Depending on the decision regarding dummy records, we might want to exclude them here as well, just in case.

Reply:
they are already excluded when dumping but good point